pkg/disk/cloud.go

```diff
@@ -521,6 +521,24 @@ func getDisk(diskID string, ecsClient *ecs.Client) []ecs.Disk {
 	return diskResponse.Disks.Disk
 }
 
+func tagDiskUserTags(diskID string, ecsClient *ecs.Client, tags map[string]string) {
+	addTagsReq := ecs.CreateAddTagsRequest()
+	var userTags []ecs.AddTagsTag
+	for k, v := range tags {
+		userTags = append(userTags, ecs.AddTagsTag{Key: k, Value: v})
+	}
+	addTagsReq.Tag = &userTags
+	addTagsReq.ResourceType = "disk"
+	addTagsReq.ResourceId = diskID
+	addTagsReq.RegionId = GlobalConfigVar.Region
+	_, err := ecsClient.AddTags(addTagsReq)
```

Comment:
Need to call `getEcsClientByID` before calling ECS OpenAPI, to support multi-tenant use case, or refresh the credential.

Reply:
okay